code/pyth/client/client.en.ts

```diff
@@ -0,0 +1,21 @@
+import * as web3 from "@solana/web3.js"
+import * as pyth from "@pythnetwork/client"
+
+(async () => {
+  const connection = new web3.Connection(
+    web3.clusterApiUrl('devnet'),
+    'confirmed',
+  );
+const pythConnection = new pyth.PythConnection(connection,pyth.getPythProgramKeyForCluster('devnet'));
```

Comment:
One last nit and we'll be ready to merge. Could you fix the tab formatting in this file to match elsewhere?

Reply:
Sure @jacobcreech

Reply:
Looks good. I have it as an issue to add linting to the CI and build, but haven't gotten to it yet https://github.com/solana-developers/solana-cookbook/issues/280

Reply:
Would try on that after i get the next PR done